content/docs/0.16.x/operate/advanced_install_options/index.md

```diff
@@ -1,11 +1,9 @@
 ---
-title: Advanced Install Options
+title: Install Keptn using the Helm chart
```

Comment:
Again, replacing this title is in the plans and, since this text links to that file, I want to replace the title.  This one seems particularly critical since we are on the brink of making the Helm chart the only installation option so we do not want new users thinking of this as an "advanced option".

Reply:
You said to restore the old Quick Start without modifications so that is what I did.  Do you instead want me to add a brief statement to the Quick Start that this gives instructions for installing Keptn on a local k8s cluster and running some exercises, and link to the new stuff as a way to explore Keptn without having to install anything locally?